eZ/Publish/Core/Repository/LocationService.php

```diff
@@ -1000,6 +1009,8 @@ private function checkCreatePermissionOnSubtreeTarget(APILocation $targetParentL
                         [
                             new CriterionSubtree($loadedSubtree->pathString),
```

Comment:
Looks like we already have:
```suggestion
                            new CriterionSubtree($loadedSubtree->getPathString()),
```
so it would be nice side performance fix.